pkgs/development/libraries/glibc/common.nix

```diff
@@ -153,6 +153,8 @@ stdenv.mkDerivation ({
       "--enable-add-ons"
       "--sysconfdir=/etc"
       "--enable-stackguard-randomization"
+      "--enable-static-pie"
+      "--enable-bind-now"
```

Comment:
Static pie needs this. It is also a security hardening know as full-relo: https://ctf101.org/binary-exploitation/relocation-read-only/

This is know close to what Archlinux has in its gcc.

Reply:
It looks like our hardening had `bindnow` and `relro` enabled anyway for most binaries.